app/components/story/StoryCard.js

```diff
@@ -4,18 +4,48 @@ import {
   Text,
   StyleSheet,
   Dimensions,
+  Animated,
 } from 'react-native';
 
 class StoryCard extends Component {
+  constructor(props) {
+    super(props); 
+    this.state = {
+      height: new Animated.Value() 
+    };
+  }
+
+  componentWillMount() {
```

Comment:
without this, the height of the top card will always be set to the minimum height needed to display all of the top text (this will change as you swipe right and change the top text content). also, the top and bottom text will always be shown.